calicoctl/calico_ctl/node.py

```diff
@@ -391,13 +391,20 @@ def _start_node_container_docker(ip, ip6, as_num, log_dir, node_image, detach, e
         if err.response.status_code != 404:
             raise
 
+    # This is to convert libnetwork_enabled (bool) into a string to pass it as an ENV var value
+    if libnetwork_enabled:
+       libnetwork_flag_str = "true"
+    else:
+       libnetwork_flag_str = "false"
+
     environment = [
         "HOSTNAME=%s" % hostname,
         "IP=%s" % (ip or ""),
         "IP6=%s" % (ip6 or ""),
         "CALICO_NETWORKING_BACKEND=%s" % backend,
         "AS=%s" % (as_num or ""),
-        "NO_DEFAULT_POOLS=%s" % (no_default_pools or "")
+        "NO_DEFAULT_POOLS=%s" % (no_default_pools or ""),
+        "CALICO_LIBNETWORK_ENABLED=%s" % libnetwork_flag_str
```

Comment:
python doesn't seem to have a `%b`, it has `%r` to format to binary `True` or `False` but it won't be consistent with the ENV vars we are setting (in all lower case). See flag `CALICO_NETWORKING` for example which also has to converted into `false` (string) from `False`(bool)